src/screens/HomeScreen.js

```diff
@@ -215,6 +239,7 @@ export const HomeScreen = ({ navigation, route }) => {
     // this will trigger the onRefresh functions provided to the `useHomeRefresh` hook in other
     // components.
     DeviceEventEmitter.emit(HOME_REFRESH_EVENT);
+    contentListRefetch();
```

Comment:
please put a new line in between as this has nothing to do with the prior comment about `useHomeRefresh`. best case would be adding an own comment about `contentListRefetch` here

Reply:
https://github.com/smart-village-solutions/smart-village-app-app/pull/958/commits/c5071d1275e9c9450bb63ab91c3c2de325370162